blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sourcegraph 3.31 release"
+publishDate: 2021-08-20T10:00-07:00
+description: "Sourcegraph 3.31 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.31"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+  - description: "Backend Code Insights GraphQL queries now support arguments `includeRepoRegex` and `excludeRepoRegex` to filter on repository names."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23256
+    category: Code Insights
+  - description: Operator documentation has been added to the Search Reference sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23116
+    category: Search
+  - description: Syntax highlighting support for the Cue language.
+    url: https://cuelang.org
+    category: Search
+  - description: Reintroduced a revised version of the Search Types sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23170
+    category: Search
+  - description: "Added support for [`git p4`'s `--use-client-spec` option](https://git-scm.com/docs/git-p4#Documentation/git-p4.txt---use-client-spec) can now be enabled by configuring the `p4.client` field in the Perforce integration."
```

Comment:
@iskyOS - we can't embed links in the changelog (for some reason, it doesn't work). Can you clarify this update as well?

Reply:
Yes!